pkg/ipmi/odm/ocp.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-package wiwynn
+package ocp
```

Comment:
Fair -- I think I went just by looking at the oenmap when I suggested "wiwynn" as the name.

Can you make the directory name "ocp"? Or make the package name "odm"? Generally, package name == directory name in Go.

(What does "odm" stand for?)

Reply:
@insomniacslk, I've revised the file name and folder name.  As @hugelgupf mentioned, It seems package name has to be the same as directory name in Go. Do you have any idea about it?

Reply:
I think odm cam be vendor-specific here.

Reply:
It doesn't have to match, but most Go projects require them to match for readability.

Otherwise it's hard to match "ocp.GetOemIpmiDimmInfo" in systemboot to the import statement of "pkg/ipmi/odm", unless you happen to know that odm contains ocp.

Reply:
"odm" stands for Original Design Manufacturer - @timchuquanta and I agreed upfront  to put ODM-specific code under `pkg/ipmi/odm` instead of actual vendor names (or we would end up with one package per ODM, and duplicate code). The structure I suggested was:
```
pkg/
└── ipmi
    └── odm
        └── ocp.go
```

(in other words, the same as the advice from @hugelgupf )